CharlieBackend.Api/Controllers/MentorsController.cs

```diff
@@ -41,6 +41,10 @@ public MentorsController(IMentorService mentorService, ILessonService lessonServ
         [HttpPost("{accountId}")]
         public async Task<ActionResult> PostMentor(long accountId)
         {
+            if (!ModelState.IsValid)
```

Comment:
https://github.com/ita-social-projects/WhatBackend/pull/519#discussion_r657413864
@andrii-developer is right. You don't need to manually check the ModelState in Web API controller.

>Web API controllers don't have to check `ModelState.IsValid` if they have the `[ApiController]` attribute. In that case, an automatic HTTP 400 response containing error details is returned when model state is invalid.
[Source: Model state section](https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation#model-state)